bin/build_container_image

```diff
@@ -11,7 +12,7 @@ docker build . -t $IMAGE_NAME
 
 [[ $TRAVIS_PULL_REQUEST != "false" ]] && exit 0
 
-docker login $REGISTRY -u $REGISTRY_USERNAME -p $REGISTRY_PASSWORD
+echo $REGISTRY_PASSWORD | docker login $REGISTRY -u $REGISTRY_USERNAME --password-stdin
```

Comment:
Minor, but we should quote this just in case it has some special characters in it.

```suggestion
echo "$REGISTRY_PASSWORD" | docker login $REGISTRY -u $REGISTRY_USERNAME --password-stdin
```